services/gr-css-reports/launch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+function retry () {
+  for _ in {1..9}; do
+    "$@" && return
+    sleep 30
+  done
+  "$@"
+}
```

Comment:
`source ~worker/.local/bin/common.sh`  for `retry` instead. It handles pipelines better in case retry actually occurs. (so `jshon` will only get stdout from the successful call, if a prior one output anything before it failed ... this has happened).